src/main/java/io/bloco/faker/Faker.java

```diff
@@ -52,6 +52,8 @@ public Faker(String locale) {
         this.book = this.data.getComponent(Book.class);
         this.company = this.data.getComponent(Company.class);
         this.name = this.data.getComponent(Name.class);
+        // TODO: CHANGE THIS??
+        this.time = new Time();
```

Comment:
I instantiated the componentes in the FakerData constructor. Then you can use this.data.getComponent to find it.